examples/rbd/block-pod-clone.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: pod-with-raw-block-volume-clone
```

Comment:
same here, no need to include `raw` in the name

Reply:
same as above, will take care of it in next PR.